x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/index.tsx

```diff
@@ -73,12 +62,15 @@ export const HostList = () => {
   const onTableChange = useCallback(
     ({ page }: { page: { index: number; size: number } }) => {
       const { index, size } = page;
-      dispatch({
-        type: 'userPaginatedHostList',
-        payload: { pageIndex: index, pageSize: size },
-      });
+      history.push(
+        urlFromQueryParams({
+          ...queryParams,
+          page_index: JSON.stringify(index),
```

Comment:
(optional) You don't need stringify here and below

Reply:
because the type is actually set up as string, typescript complains if i don't have stringify it here. maybe i should just turn that into a number and all will be solved?!